#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE = "tarkbot_robot"

gen = ParameterGenerator()

gen.add("imu_calibrate", bool_t, 0, "Check to start the calibration", False)

gen.add("light_calibrate", bool_t, 0, "save the change about light", False)

gen.add("RGB_M",int_t,0,"int RGB_M",1,1,10)
gen.add("RGB_S",int_t,0,"int RGB_S",1,1,10)
gen.add("RGB_T",int_t,0,"int RGB_T",0,0,255)

gen.add("RGB_R",int_t,0,"int RGB_R",0,0,255)
gen.add("RGB_G",int_t,0,"int RGB_G",255,0,255)
gen.add("RGB_B",int_t,0,"int RGB_B",0,0,255)

exit(gen.generate(PACKAGE, "tarkbot_robot", "robot"))
